vtex-apps/order-manager
# CHK-611: Use @vtex/order-manager

#### What problem is this solving?

The aim of this PR is to make use of the new `@vtex/order-manager` package.  



#### How should this be manually tested?

- Use this [Workspace](https://ompackage--checkoutio.myvtex.com/cart/add?sku=289)
- Check if the cart information is presented to you


#### Checklist/Reminders


#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/12574426/113311325-916af400-92df-11eb-9ac4-0695fdd6cce2.png)


#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements
